transaction-platform-tck/pom.xml

```
@@ -0,0 +1,377 @@
+                            <sqlCommand>CREATE SCHEMA CTS1 AUTHORIZATION CTS1;</sqlCommand>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>transactions-platform-tck</id>
+                        <goals>
+                            <goal>verify</goal>
+                            <goal>integration-test</goal>
+                        </goals>
+                        <configuration>
+                            <dependenciesToScan>jakarta.tck:transactions-tck</dependenciesToScan>
+                            <groups>${jakarta.javatest.group}</groups>
+
+                            <includes>
+                                <include>com/sun/ts/tests/jta/ee/transactional/ClientEjblitejspTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/transactional/ClientEjbliteservletTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/transactional/ClientEjbliteservlet2Test*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/transactional/ClientEjblitejsfTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/txpropagationtest/ClientServletTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/txpropagationtest/ClientJspTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/usertransaction/**/*JspTest*.java</include>
+                                <include>com/sun/ts/tests/jta/ee/usertransaction/**/*ServletTest*.java</include>
+                            </includes>
+
+                            <systemPropertyVariables>
+                                <payara.home>${payara.home}</payara.home>
+                                <arquillian.launch>tck-javatest</arquillian.launch>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>appclient</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>002-copy-lib</id>
+                                <goals>
+                                    <goal>copy</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <artifactItems>
+
```

Skip wouldn't go amiss for remote profile
```suggestion
                                <configuration>
                                    <skip>${skipConfig}</skip>
                                    <artifactItems>
```